modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/resources.jsp

```diff
@@ -35,6 +35,34 @@ String inlineEditSaveURL = GetterUtil.getString((String)request.getAttribute(CKE
 	long javaScriptLastModified = PortalWebResourcesUtil.getLastModified(PortalWebResourceConstants.RESOURCE_TYPE_EDITOR_CKEDITOR);
 	%>
 
+	<script data-senna-track="temporary" type="text/javascript">
```

Comment:
(Non-blocking feedback; change it if you want, or don't.)

You don't need `type="text/javascript"`. From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script):

> The HTML5 specification urges authors to omit the attribute rather than provide a redundant MIME type.

I know you'll find plenty of examples sprinkled around liferay-portal, but you will also find the opposite (eg. `<script>`).

Reply:
I know I don't need it but as you've guessed I've added it here because the rest of the script tags in that jsp file had this attribute, but I agree: if we don't need it let's not add it.